bevy_rapier3d/examples/testbed3.rs

```diff
@@ -1,283 +1,284 @@
-#![allow(dead_code)]
+// FIXME: Uncomment plugins once `bevy_egui` and `bevy_inspector_egui` get updated to be compatible with bevy 0.16
```

Comment:
Leaving a comment to bring attention to this ; it's been discussed in main PR thread already :+1: